development/extract_strings_from_objects.py

```diff
@@ -161,12 +161,13 @@ def add_line(value, component, sep, key, filename):
 			text = u'_("{value}")'.format(value=value[2:])
 			component = component + sep + str(parse_token(key, 'TIER'))
 			filename = filename.rsplit('.yaml')[0].split(OBJECT_PATH)[1].replace('/',':')
-			comment = '%s of %s' %(component, filename)
-			object_strings.append('# %s' %comment + ROWINDENT + '%-30s: %s' % (('"%s"') % component, text))
+			comment = u'{0!s} of {1!s}'.format(component, filename)
+			object_strings.append('# {comment!s}{rowindent}{component:<30}: {text!s}'.
+				format(comment=comment, rowindent=ROWINDENT, component='"{}"'.format(component), text=text))
```

Comment:
I'd drop the `component='"{}"'.format(component)` (replace it with `component=component` and instead include the quotes in the original string.

`'# {comment!s}{rowindent}"{component:<30}": {text!s}'`